index.js

```diff
@@ -1,10 +1,5 @@
 
 export { fetchingData } from './src/artemisa/component'
-import { artemisa } from './src/artemisa/reducer'
-import { dataService } from './src/core/service'
-
-export const reducer = artemisa
-export const service = dataService
-
-export { isFetching, isFetched } from './src/core/reducer'
+export { artemisa as reducer, isFetching, isFetched } from './src/artemisa/reducer'
+export { dataService as service } from './src/core/service'
```

Comment:
better expressed exports with aliases.
Removing `core/reducer` exports since that's part of what I cleaned.
`core/reducer` was first thought to be used by end-user, decorating their reducers with that one, to store the fetched value in the user store.
That is not the idea anymore, artemisa aims to manage its own store tree for the cache, end-user don't need to store it in their own store tree, since artemisa automatically injects the data into the component, also avoiding the need for connect containers